modules/core/src/effects/effect.js

```diff
@@ -0,0 +1,14 @@
+export default class Effect {
+  constructor(props = {}) {
+    const {id = 'effect'} = props;
+    this.id = id;
+    this.props = {};
+    Object.assign(this.props, props);
+  }
+
+  setProps(props) {
```

Comment:
Can you add API audit notes in a tracker task: 
* Allowing `setProps` on effects makes their API quite different from layers.